dockerfiles/README.md

```diff
@@ -22,8 +22,8 @@ Use `docker pull` with any of the images and tags below to pull an image and try
 
 | Build Flavor      | Base Image                            | ONNX Runtime Docker Image tags        | Latest                  |
 |-------------------|---------------------------------------|---------------------------------------|-------------------------|
-| Source (CPU)      | mcr.microsoft.com/azureml/onnxruntime | :v0.4.0, :v0.5.0, v0.5.1, :v1.0.0, :v1.2.0, :v1.3.0     | :latest                 |
-| CUDA (GPU)        | mcr.microsoft.com/azureml/onnxruntime | :v0.4.0-cuda10.0-cudnn7, :v0.5.0-cuda10.1-cudnn7, :v0.5.1-cuda10.1-cudnn7, :v1.0.0-cuda10.1-cudnn7, :v1.2.0-cuda10.1-cudnn7, :v1.3.0-cuda10.1-cudnn7 | :latest-cuda            |
+| Source (CPU)      | mcr.microsoft.com/azureml/onnxruntime | :v0.4.0, :v0.5.0, v0.5.1, :v1.0.0, :v1.2.0, :v1.3.0, :v1.4.0     | :latest                 |
+| CUDA (GPU)        | mcr.microsoft.com/azureml/onnxruntime | :v0.4.0-cuda10.0-cudnn7, :v0.5.0-cuda10.1-cudnn7, :v0.5.1-cuda10.1-cudnn7, :v1.0.0-cuda10.1-cudnn7, :v1.2.0-cuda10.1-cudnn7, :v1.3.0-cuda10.1-cudnn7, :v1.4.0-cuda10.1-cudann7 | :latest-cuda            |
 | TensorRT (x86)    | mcr.microsoft.com/azureml/onnxruntime | :v0.4.0-tensorrt19.03, :v0.5.0-tensorrt19.06, :v1.0.0-tensorrt19.09, :v1.2.0-tensorrt20.01, :v1.3.0-tensorrt20.01 | :latest-tensorrt        |
```

Comment:
can you add v1.4.0-tensorrt20.01 tag as well? 
I just published it.

Reply:
published openvino images as well. so the following tags can be added as well
```
v1.4.0-openvino-2020.3.194-cpu
v1.4.0-openvino-2020.3.194-gpu
v1.4.0-openvino-2020.3.194-myriad
v1.4.0-openvino-2020.3.194-vadm
```

Reply:
Have you also advanced latest tags for them?

---
In reply to: [458782750](https://github.com/microsoft/onnxruntime/pull/4574#discussion_r458782750) [](ancestors = 458782750)

Reply:
@jywu-msft I have removed VAD-M installation instruction that pointed to 2020.2 version